test/upgrade/prober/sender.go

```diff
@@ -88,8 +94,33 @@ func (p *prober) deploySender() {
 func (p *prober) removeSender() {
 	p.log.Info("Remove of sender deployment: ", senderName)
 
+	foreground := metav1.DeletePropagationForeground
+	dOpts := metav1.DeleteOptions{
+		GracePeriodSeconds: pointer.Int64(0),
```

Comment:
/hold

Reply:
https://github.com/knative/eventing/blob/2d64fdd97d1ad3aade6efa4c58a4d5cee9977f87/test/upgrade/prober/wathola/sender/services.go#L56-L66

Reply:
hmm, `terminationGracePeriodSeconds` and delete grace period should not be the same thing.

Reply:
We don't need gracePeriodSeconds since we have the wait loop to make sure that the sender is deleted.